{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":97,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":101,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":105,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":107,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":122,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":141,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":148,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":155,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":162,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":166,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":179,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":182,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":185,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":196,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":210,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":217,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":238,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":281,"column":1}}},"severity":"INFO"}
{"message":"Using failwith unsafely","location":{"path":"Ruby/lib/interpreter.ml","range":{"start":{"line":294,"column":1}}},"severity":"INFO"}
